script.js

```diff
@@ -1,4 +1,122 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  // setting variables to avoid typo
+  var PAPER_OPTION = "paper";
+  var SCISSOR_OPTION = "scissor";
+  var STONE_OPTION = "stone";
+
+  // seetting variables for reverse SPS Game
+
+  var REV_STONE = "reversed stone";
+  var REV_PAPER = "reversed paper";
+  var REV_SCISSOR = "reversed scissor";
+
+  var generateComputerOption = function () {
+    var RandomNo = Math.random() * 3;
+    var randomSelection = Math.ceil(RandomNo);
+
+    if (randomSelection == 1) {
+      return SCISSOR_OPTION;
+    }
+    if (randomSelection == 2) {
+      return PAPER_OPTION;
+    }
+
+    return STONE_OPTION;
+  };
```

Comment:
This function can be placed outside of the main function to declutter the main function.